clang/include/clang/CIR/Dialect/IR/CIROps.td

```diff
@@ -5214,4 +5214,29 @@ def SignBitOp : CIR_Op<"signbit", [Pure]> {
   }];
 }
 
+//===----------------------------------------------------------------------===//
+// StdInitializerListOp
+//===----------------------------------------------------------------------===//
+
+def StdInitializerListOp : CIR_Op<"std_initializer_list.ctor"> {
+  let summary = "Initialize std::initializer_list";
+  let description = [{
+    The `std_initializer_list.ctor` operation will initialize
+    `std::initializer_list<T>` with given arguments list.
+
+    ```cpp
+    initializer_list<int> v{1,2,3}; // initialize v with 1, 2, 3
+    ```
```

Comment:
Thanks, a C++ example is nice. But it would be nicer to further show how this line of code is modelled in ClangIR with your new operation.